src/common/responses.py

```diff
@@ -98,7 +99,7 @@ async def wrapper_decorator(*args, **kwargs) -> TResponse | JSONResponse:
             except ValidationException as e:
                 if logger.level <= logging.DEBUG:
                     traceback.print_exc()
-                logger.error(e)
+                logger.error(e, extra={"Traceback": get_traceback()})
```

Comment:
Consider if this can be skipped.  Traceback will log user-input, and ValidationException "should" be easy to provide the user with a readable error message?